boom! testing git-workarea-1.0.1 against stable for nightly
boom! running: cargo +stable build --frozen
boom! creating container for: cargo +stable build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable build --frozen cargobomb`
blam! 319972b5d3fd9237ea6266c5f7c6d80ed0ed79b6d2025949a9027b0e48a5a325
boom! running `docker start -a 319972b5d3fd9237ea6266c5f7c6d80ed0ed79b6d2025949a9027b0e48a5a325`
kablam!    Compiling git-workarea v1.0.1 (file:///source)
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 3.8 secs
boom! running `docker rm -f 319972b5d3fd9237ea6266c5f7c6d80ed0ed79b6d2025949a9027b0e48a5a325`
blam! 319972b5d3fd9237ea6266c5f7c6d80ed0ed79b6d2025949a9027b0e48a5a325
boom! running: cargo +stable test --frozen --no-run
boom! creating container for: cargo +stable test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen --no-run cargobomb`
blam! e01ec3d434318d74ced080907c17681745ffd042c3caa324ec5c7fc921932c5b
boom! running `docker start -a e01ec3d434318d74ced080907c17681745ffd042c3caa324ec5c7fc921932c5b`
kablam!    Compiling git-workarea v1.0.1 (file:///source)
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 5.1 secs
boom! running `docker rm -f e01ec3d434318d74ced080907c17681745ffd042c3caa324ec5c7fc921932c5b`
blam! e01ec3d434318d74ced080907c17681745ffd042c3caa324ec5c7fc921932c5b
boom! running: cargo +stable test --frozen
boom! creating container for: cargo +stable test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/stable:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/stable:/target:rw -e USER_ID=500 -e CMD=cargo +stable test --frozen cargobomb`
blam! 8f63625db7a536065ddd24aa87bd7878a4f758384de6a022c2653c5d64c8420a
boom! running `docker start -a 8f63625db7a536065ddd24aa87bd7878a4f758384de6a022c2653c5d64c8420a`
kablam!     Finished debug [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/git_workarea-e01d52a11ba3d69c
blam! 
blam! running 19 tests
blam! test test::test_gitdir ... FAILED
blam! test test::test_mergeable ... FAILED
blam! test test::test_mergeable_already_merged ... FAILED
blam! test test::test_mergeable_no_common_history ... FAILED
blam! test test::test_mergeable_conflict ... FAILED
blam! test test::test_reserve_ref_no_exist ... FAILED
blam! test test::test_setup_merge_conflict ... FAILED
blam! test test::test_reserve_ref ... FAILED
blam! test test::test_setup_merge ... FAILED
blam! test test::test_reserve_refs ... FAILED
blam! test test::test_setup_merge_rewind ... FAILED
blam! test test::test_setup_merge_submodule_not_merged ... FAILED
blam! test test::test_setup_merge_submodule_not_merged_conflict ... FAILED
blam! test test::test_setup_merge_submodule_conflict_resolution ... FAILED
blam! test test::test_work_area_setup_no_exist ... FAILED
blam! test test::test_work_area_setup ... FAILED
blam! test test::test_work_area_setup_submodule ... FAILED
blam! test test::test_setup_merge_submodule_not_present ... FAILED
blam! test test::test_work_area_setup_submodule_missing_links ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- test::test_gitdir stdout ----
blam! 	thread 'test::test_gitdir' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- test::test_mergeable stdout ----
blam! 	thread 'test::test_mergeable' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_mergeable_already_merged stdout ----
blam! 	thread 'test::test_mergeable_already_merged' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_mergeable_no_common_history stdout ----
blam! 	thread 'test::test_mergeable_no_common_history' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_mergeable_conflict stdout ----
blam! 	thread 'test::test_mergeable_conflict' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_reserve_ref_no_exist stdout ----
blam! 	thread 'test::test_reserve_ref_no_exist' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_setup_merge_conflict stdout ----
blam! 	thread 'test::test_setup_merge_conflict' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_reserve_ref stdout ----
blam! 	thread 'test::test_reserve_ref' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_setup_merge stdout ----
blam! 	thread 'test::test_setup_merge' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_reserve_refs stdout ----
blam! 	thread 'test::test_reserve_refs' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_setup_merge_rewind stdout ----
blam! 	thread 'test::test_setup_merge_rewind' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_setup_merge_submodule_not_merged stdout ----
blam! 	thread 'test::test_setup_merge_submodule_not_merged' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_setup_merge_submodule_not_merged_conflict stdout ----
blam! 	thread 'test::test_setup_merge_submodule_not_merged_conflict' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_setup_merge_submodule_conflict_resolution stdout ----
blam! 	thread 'test::test_setup_merge_submodule_conflict_resolution' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_work_area_setup_no_exist stdout ----
blam! 	thread 'test::test_work_area_setup_no_exist' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_work_area_setup stdout ----
blam! 	thread 'test::test_work_area_setup' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_work_area_setup_submodule stdout ----
blam! 	thread 'test::test_work_area_setup_submodule' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_setup_merge_submodule_not_present stdout ----
blam! 	thread 'test::test_setup_merge_submodule_not_present' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_work_area_setup_submodule_missing_links stdout ----
blam! 	thread 'test::test_work_area_setup_submodule_missing_links' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! 
blam! failures:
blam!     test::test_gitdir
blam!     test::test_mergeable
blam!     test::test_mergeable_already_merged
blam!     test::test_mergeable_conflict
blam!     test::test_mergeable_no_common_history
blam!     test::test_reserve_ref
blam!     test::test_reserve_ref_no_exist
blam!     test::test_reserve_refs
blam!     test::test_setup_merge
blam!     test::test_setup_merge_conflict
blam!     test::test_setup_merge_rewind
blam!     test::test_setup_merge_submodule_conflict_resolution
blam!     test::test_setup_merge_submodule_not_merged
blam!     test::test_setup_merge_submodule_not_merged_conflict
blam!     test::test_setup_merge_submodule_not_present
blam!     test::test_work_area_setup
blam!     test::test_work_area_setup_no_exist
blam!     test::test_work_area_setup_submodule
blam!     test::test_work_area_setup_submodule_missing_links
blam! 
blam! test result: FAILED. 0 passed; 19 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f 8f63625db7a536065ddd24aa87bd7878a4f758384de6a022c2653c5d64c8420a`
blam! 8f63625db7a536065ddd24aa87bd7878a4f758384de6a022c2653c5d64c8420a
